### ISSUE TITLE
Option to set "reconnect false;" for shiny apps

### ISSUE THREAD

**Issue by vnijs** [jupyterhub/jupyter-rsession-proxy#72]
shiny-server has a configuration option called `reconnect` (see https://community.rstudio.com/t/app-feature-works-on-shinyapps-io-but-not-with-shiny-server-pro/36285/3). Is there an option to set this to `false` for Shiny apps launched using the Shiny icon via `jupyter-rsession-proxy`?

Related question: Does jupyter have a similar `reconnect` option to shiny-server I could turn off as shiny apps launched from Rstudio through jupyterlab seem to mess up the same feature mentioned in the post above.

**Comment by ryanlovett** [comment]
This extension doesn't let you alter the shiny-server configuration [except for a few parameters](https://github.com/jupyterhub/jupyter-rsession-proxy/blob/master/jupyter_rsession_proxy/__init__.py#L13), though it could be extended to do so.

[According to the docs](https://docs.rstudio.com/shiny-server/#local-app-configurations), `reconnect` can be specified in local app configurations, but that would require adding `allow_app_override` in the global config. That makes sense for this extension and would be straightforward to add.

Not sure about the latter issue.

**Comment by vnijs** [comment]
Thanks for the response @ryanlovett. I ended up using the source from the [1.0b6 build](https://pypi.org/project/jupyter-rsession-proxy/) because I couldn't get the source from master to work with my docker container. I made a few changes and they (mostly) work as intended. See link below:

https://github.com/vnijs/jupyter-rsession-proxy/blob/master/jupyter_rsession_proxy/__init__.py

The one weird thing I still get is that my shiny app "hangs" on browser refresh in Rstudio-server when started from jupyterlab. Works fine with RStudio-server called directly. If you have any ideas/suggestions on what might be going on I'd love to hear them. Else, feel free to close this issue.
